docs/gatewify-an-app.md

```diff
@@ -0,0 +1,169 @@
+# Gateway 
+
+![gateway](https://github.com/georchestra/georchestra-gateway/actions/workflows/docker.yml/badge.svg)
+
+The gateway belongs to geOrchestra core, since it is the component which :
+
+* handles user sessions
+* routes requests to webapps
+
+The behavior is controlled by the files from the `<datadir_root>/security-proxy` folder, which can be found [here](https://github.com/georchestra/datadir/tree/master/security-proxy)
+
+## How-to integrate a new application in geOrchestra ?
+
+The goal here is to benefit from the [SSO](https://en.wikipedia.org/wiki/Single_sign-on) feature for the new application without having to use an external application.
```

Comment:
"to use an external authentication process" ? instead of "an external application".